### PR TITLE
[homework#3] HTML-CSS-CV by Roman Palagniuk

### PR DESCRIPTION
**Pull request by RomanPalagniuk**
Screenshot_2
Screenshot_4
Screenshot_5

### CODE REVIEW

File: homeworks/roman-palagniuk/03-html-css/cv.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
```
Comment (solvd-mmelnyk) [inline homeworks/roman-palagniuk/03-html-css/cv.css:7]: этот комент касается всех файлов в этом PR
между некоторыми отступы две линии, между другими одна
давай придерживаться какого-то стайл, поставь может себе prettier

File: homeworks/roman-palagniuk/03-html-css/cv.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+h2, p {
+    margin: 1rem 0 ;
+}
+
+h1 {
+    text-align: center;
+}
+
+
```
Comment (solvd-mmelnyk) [inline homeworks/roman-palagniuk/03-html-css/cv.css:28]: the same as above ^^

File: homeworks/roman-palagniuk/03-html-css/cv.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    <link rel="stylesheet" href="./cv.css">
+    <title>CV</title>
+</head>
+<body>
+    <header>
+        <i style="font-size:48px; float: left; background-color: khaki; color:  rgb(59, 55, 20)" class="fa fa-user-secret">&#xf2be;</i>
+        <h1>
+            Roman Palagniuk
+        </h1>        
+
```
Comment (solvd-mmelnyk) [inline homeworks/roman-palagniuk/03-html-css/cv.html:17]: what purpose of this empty string?

File: homeworks/roman-palagniuk/03-html-css/index.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
```
Comment (solvd-mmelnyk) [inline homeworks/roman-palagniuk/03-html-css/index.css:6]: please make sure that styleguid also applied for this file

File: homeworks/roman-palagniuk/03-html-css/cv.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w==" crossorigin="anonymous" />
+    <link rel="stylesheet" href="./cv.css">
+    <title>CV</title>
+</head>
+<body>
+    <header>
+        <i style="font-size:48px; float: left; background-color: khaki; color:  rgb(59, 55, 20)" class="fa fa-user-secret">&#xf2be;</i>
```
Comment (solvd-mmelnyk) [inline homeworks/roman-palagniuk/03-html-css/cv.html:13]: why did you use inline styles here?

**Review by solvd-mmelnyk — CHANGES_REQUESTED** [review]

**Comment by solvd-mmelnyk** [comment]
@RomanPalagniuk 
Please make sure that all your HTML and CSS should pass W3C HTML and CSS validations

File: homeworks/roman-palagniuk/03-html-css/cv.css (hunk 1 above)
Comment (RomanPalagniuk) [reply homeworks/roman-palagniuk/03-html-css/cv.css:7]: have Prettier, maybe it's not working or I should configure it right way

File: homeworks/roman-palagniuk/03-html-css/cv.html (hunk 5 above)
Comment (RomanPalagniuk) [reply homeworks/roman-palagniuk/03-html-css/cv.html:13]: part of the task was to use different ways of styles conection. so I used inline for example. know it's a bad practice.
was checking files with validators.

Comment (solvd-mmelnyk) [reply homeworks/roman-palagniuk/03-html-css/cv.html:13]: @RomanPalagniuk yes, you are right
you can leave these styles as inline

**Pull request closed without merging**